Firestore/Example/FuzzTests/FSTFuzzTestsPrincipal.mm

```diff
@@ -49,29 +61,76 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
   return 0;
 }
 
+// Retrieves the fuzzing target from the FUZZING_TARGET environment variable.
+// Default target is kNone if the environment variable is empty, not set, or
+// could not be interpreted. Should be kept in sync with FuzzingTarget.
+FuzzingTarget GetFuzzingTarget() {
+  char *fuzzing_target_env = std::getenv("FUZZING_TARGET");
```

Comment:
Sorry to keep nitpicking you:
* since you don't intend to change this string, this would better be `const char*`;
* the comparisons below would be much nicer if using `std::string`. It's best to avoid `strcmp` if possible. The only thing to be aware of is that constructing a `std::string` from a null pointer to char is undefined behavior. So I'd suggest turning `fuzzing_target_env` into a `std::string` immediately after checking it for `NULL` below. Then the comparisons would be just `if (fuzzing_target_env == "NONE")`, etc.

Reply:
I totally agree, I was concerned about `strcmp` as well but I thought the CR would clarify things. Thanks!